AudioBarTests/SeekForwardTests.swift

```diff
@@ -1,77 +0,0 @@
-import XCTest
-@testable import AudioBar
-
-final class SeekForwardTests: XCTestCase {
-
```

Comment:
Compared to `master`. So, I collapsed the following four tests ...

```swift
func testWhenPlaying1() {
    var model = Model.readyToPlay(.init(isPlaying: true, duration: 60, currentTime: 0))
    let commands = try! Module.update(for: .seekForward, model: &model)
    XCTAssertEqual(model, .readyToPlay(.init(isPlaying: true, duration: 60, currentTime: 15)))
    XCTAssertEqual(commands, [.player(.setCurrentTime(15))])
}

func testWhenPlaying2() {
    var model = Model.readyToPlay(.init(isPlaying: true, duration: 120, currentTime: 60))
    let commands = try! Module.update(for: .seekForward, model: &model)
    XCTAssertEqual(model, .readyToPlay(.init(isPlaying: true, duration: 120, currentTime: 75)))
    XCTAssertEqual(commands, [.player(.setCurrentTime(75))])
}

func testWhenPaused1() {
    var model = Model.readyToPlay(.init(isPlaying: false, duration: 60, currentTime: 0))
    let commands = try! Module.update(for: .seekForward, model: &model)
    XCTAssertEqual(model, .readyToPlay(.init(isPlaying: false, duration: 60, currentTime: 15)))
    XCTAssertEqual(commands, [.player(.setCurrentTime(15))])
}

func testWhenPaused2() {
    var model = Model.readyToPlay(.init(isPlaying: false, duration: 120, currentTime: 60))
    let commands = try! Module.update(for: .seekForward, model: &model)
    XCTAssertEqual(model, .readyToPlay(.init(isPlaying: false, duration: 120, currentTime: 75)))
    XCTAssertEqual(commands, [.player(.setCurrentTime(75))])
}
```

... into the following two tests ...

```swift
func testSeekForward1() {
    model = .readyToPlay(.init(isPlaying: true, duration: 60, currentTime: 0))
    message = .seekForward
    expect(model, .readyToPlay(.init(isPlaying: true, duration: 60, currentTime: 0 + 15)))
    expect(command, .player(.setCurrentTime(0 + 15)))
}

func testSeekForward2() {
    model = .readyToPlay(.init(isPlaying: false, duration: 60, currentTime: 1))
    message = .seekForward
    expect(model, .readyToPlay(.init(isPlaying: false, duration: 60, currentTime: 1 + 15)))
    expect(command, .player(.setCurrentTime(1 + 15)))
}
```

Here, `isPlaying` doesn't play any role (just like in `testSeekBack[12]`) and should be removed. Will fix.